core/src/main/java/com/graphhopper/routing/AbstractBidirectionEdgeCHNoSOD.java

```diff
@@ -159,8 +160,8 @@ protected int getOrigEdgeId(EdgeIteratorState edge, boolean reverse) {
 
     @Override
     protected int getTraversalId(EdgeIteratorState edge, int origEdgeId, boolean reverse) {
```

Comment:
The name should be also getEdgeKey and we do only need the adjNode not `edge` itself here?

Reply:
hmm for node based traversal the traversal id is the node id so I think `getTraversalId` makes sense (more general) ? But yes, let's replace the edge parameter with the adj node id.

Reply:
we cannot change the edge parameter with adj node id, because it is used by `AbstractBiDirAlgo`.

Reply:
Ah, ok, I see.